tasks/firewalld.yml

```diff
@@ -1,4 +1,14 @@
 ---
+#firewalld doesn't exist on AWS CentOS 7 by default
+- name: Make sure firewalld is installed
+  yum: name=firewalld state=present
+  when: ansible_os_family == "RedHat"
```

Comment:
I dont think we need to add the when here. other families have firewalld too, but it was a good thought